GOVERNANCE.md

```diff
@@ -1,3 +1,246 @@
-## Kuadrant Governance
+# Steering Committee Governance
 
-Governance for the Kuadrant project is still a work in progress. You can participate in discussions via our project [slack channel](https://kubernetes.slack.com/archives/C05J0D0V525).
+The Kuadrant Steering Committee is the governing body of the Kuadrant project,
+providing decision-making and oversight pertaining to the project bylaws,
+sub-organizations, and financial planning. The Steering Committee also defines
+the project values and structure.
+
+This governance is an open, living document, and will continue to
+evolve as the community and project change.
+
+## Values
+
+The Kuadrant and its leadership embrace the following values:
+
+* Openness: Communication and decision-making happens in the open and is discoverable for future
+  reference. As much as possible, all discussions and work take place in public
+  forums and open repositories.
+
+* Fairness: All stakeholders have the opportunity to provide feedback and submit
```

Comment:
```suggestion
* Fairness: All collaborators have the opportunity to provide feedback and submit
```

Reply:
This is an interesting change.
I wonder if stakeholders implies 'users' as well, while collaborators wouldn't?
cc @alexsnaps as you mentioned users in the context of voting elsewhere in the doc